src/lib/radio/radio.md

```diff
@@ -27,3 +27,7 @@ Individual radio-buttons inside of a radio-group will inherit the `name` of the
 ### Use with `@angular/forms`
 `<md-radio-group>` is compatible with `@angular/forms` and supports both `FormsModule`
 and `ReactiveFormsModule`.
+
+### Accessibility
+Radio button groups should should be marked by `aria-label` or `aria-labelledby`
```

Comment:
I would add a paragraph here, similar to checkbox, that mentions the internal `<input type="radio">`